### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by arabiu033**
## In this milestone I covered the following:

- I first chose an API on which to base the development of the web app.

## Interactions

- Home page

> - When the page loads, the web app shows a select options field to select the categories
> - Along with the category name, Display a list of the laureates.
> - When the user clicks (or taps) on a category item, the application navigates to the  details page.

- Details page

> On the details page, the web app retrieves data from the API to show detailed data about the noble winners data.
> When the user clicks on the "Home" button (<), the user navigates to the home page.

## Testing requirements

> Created unit tests for pure functions.
> Created integration tests for the application using the React Testing Library.

## Technical requirements

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- I filtered the data that I retrieved from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is deployed and accessible online (Link to the README file).

[Video Link](https://www.loom.com/share/dd7432ff91e9451a99e53366bdac71b1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,90 @@
...
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+## Show your support
 
-### Making a Progressive Web App
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## Acknowledgments
+For Design Inspiration I would like to Acknowledge
```
Comment (KDlamini) [inline README.md]: Please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the  "Acknowledgements" section.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @arabiu033 🙋‍♀️,  

Excellent work on this project! 💯 
Your project has a beautiful design, and layout, and follows the design specifications. 🎊 
There is just one issue that you still need to address on your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Very beautiful UI ( well done on the loading spinner and categories filter 🔥 )
✔️  Professional pull request
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (arabiu033) [reply README.md]: Done. Thanks :+1:

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @arabiu033,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f313149537762674378457a4d79592f67697068792e676966


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**